src/app/public/modules/adapter-service/adapter.service.ts

```diff
@@ -191,6 +199,39 @@ export class SkyCoreAdapterService {
     return false;
   }
 
+  /**
+   * Remove inline height styles from the provided elements.
+   * @param elementRef - The element to search within.
+   * @param selector - The CSS selector to use when finding elements for removing height.
+   */
+  public resetHeight(elementRef: ElementRef, selector: string): void {
+    const children = elementRef.nativeElement.querySelectorAll(selector);
+    if (children.length > 0) {
+      for (let i = 0; i < children.length; i++) {
+        // tslint:disable-next-line: no-null-keyword
+        children[i].style.height = null;
```

Comment:
Remind me, do browsers need this to be `null` instead of `undefined`? Maybe worth a quick comment?

Reply:
Yep. I added a comment.